monai/inference_instructions.md

```diff
@@ -0,0 +1,75 @@
+## Instructions for running inference with the contrast-agnostic spinal cord segmentation model
```

Comment:
is there a reason for not calling this file README.md?

I was looking for such a file to understand how to use your code (and probably most people will do)

Reply:
makes sense! working on it

Reply:
Fixed in commit https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/commit/5e3f97a31aa64b4883be7a3217090bc46aa74af2